packer/flow.go

```diff
@@ -248,6 +248,10 @@ func (f *Flow) Pack(privateKey *ecdsa.PrivateKey, newBlockConflicts uint32, shou
 		builder.COM()
 	}
 
+	if newBlockConflicts > 0 && f.PosActive {
```

Comment:
Suggestion: Maybe we can check `newBlockConflicts` within the `f.PosActive` condition above? So we can delete this one here and avoid checking twice against `f.PosActive`. The `builder` would need to be created before the first check though.